mesos-client/src/main/scala/mesosphere/mesos/examples/UselessFramework.scala

```diff
@@ -0,0 +1,62 @@
+package mesosphere.mesos.examples
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, Source}
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.mesos.client.MesosClient
+import mesosphere.mesos.conf.MesosConf
+import org.apache.mesos.v1.mesos.{Filters, FrameworkInfo}
+import org.apache.mesos.v1.scheduler.scheduler.Event
+
+import scala.util.{Failure, Success}
+
+object UselessFramework extends App with StrictLogging {
+
+  /** Run Foo framework that:
+    *  - successfully subscribes
+    *  - declines all offers.
+    *
+    *  Not much, but shows the basic idea. Good to test against local mesos.
+    *
+    */
+  override def main(args: Array[String]): Unit = {
```

Comment:
Where is this called? How about and integration test?

Reply:
I think he is probably just using it to test the code. (In the end, it should probably be a separate project, but I don't have a problem with it during the prototyping phase :-))

Reply:
What Tim said ;)

Reply:
Shouldn't it be part of `src/test` then?

Reply:
Let eat bee, let eat bee...